backend/src/main/resources/application.yaml

```diff
@@ -26,8 +26,8 @@ spring:
 keycloakBackendClient:
   realm: master
   client-id: admin-cli
-  adminusername: gyanyim
-  adminpassword: snxTZr9$5GGnS7w
+  adminusername: admin
```

Comment:
Ezt ne írd át, ha importálod a közös realm-et akkor nem elvileg mennie kell a gyanyim-mal is